### DISCUSSION TITLE
If bio fetch xxx is used with head at the same time, an error will be reported

### DISCUSSION THREAD

**Discussion by ChronoWeaver** [biostars/biostar-handbook-issues#245] [Q&A]
“bio fetch NC_045512  ”
LOCUS       NC_045512              29903 bp ss-RNA     linear   VRL 18-JUL-2020
DEFINITION  Severe acute respiratory syndrome coronavirus 2 isolate Wuhan-Hu-1,
            complete genome.
...

it will work fine

but

"bio fetch NC_045512 | head"
LOCUS       NC_045512              29903 bp ss-RNA     linear   VRL 18-JUL-2020
DEFINITION  Severe acute respiratory syndrome coronavirus 2 isolate Wuhan-Hu-1,
            complete genome.
..

it will also work fine，but It will also run fine, but with the following prompt.

 /Users/guozhongfang/miniconda3/envs/bioinfo/lib/python3.8/multiprocessing/resource_tracker.py:216: UserWarning: resource_tracker: There appear to be 1 leaked semaphore objects to clean up at shutdown
  warnings.warn('resource_tracker: There appear to be %d '

So I won't be able to execute the next command，I don't know why this is.
Hope to find a solution to the problem here. (M1 Macbook pro)

**Comment by ialbert** [comment]
This is an official bug in Python,  when you Google the error you will find many instances of people getting it. But not everyone sees it.

https://github.com/python/cpython/issues/89372

It only appears when you interrupt the stream (head/tail etc) but  it is very annoying indeed,

But it does not affect the data other than printing the distracting message. 

There may be workarounds by updating certain libraries but since I can't reproduce I don't have a good test case.

**Reply by ChronoWeaver** [reply]
Thanks for your prompt reply, then I'll put this question aside for now, thanks again.
